Vagrantfile

```diff
@@ -0,0 +1,48 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Ansible, Chef, Docker, Puppet and Salt are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  # config.vm.provision "shell", inline: <<-SHELL
+  #   apt-get update
+  #   apt-get install -y apache2
+  # SHELL
+	config.vm.network "forwarded_port", guest: 5000, host: 5000
+	config.vm.provision "shell", privileged: false, inline: <<-SHELL     
+		sudo apt-get update 
+		#install pre-req
+		sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+		libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+		xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+		
+		#apt-get install pyenv as vagrant, not root
```

Comment:
A lot of commented lines in this file can be deleted. This is an example of one that looks out of date and is somewhat misleading.